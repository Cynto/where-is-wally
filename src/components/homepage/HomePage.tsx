import { Link } from 'react-router-dom';
import 'firebase/firestore';
import 'firebase/auth';
import './homepage.css';


function HomePage(props: any) {
  return (
    <div className="homepage">
      <div className="button-container">
        <Link to="/game">
          <button className="css-button-3d--sky">Play Game</button>
        </Link>
        <Link to="/leaderboard">
          <button className="css-button-3d--red">Leaderboard</button>
        </Link>
      </div>
    </div>
  );
}

export default HomePage;
