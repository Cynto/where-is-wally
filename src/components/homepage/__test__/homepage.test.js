import React from 'react';
import HomePage from '../HomePage';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';

describe('Homepage tests', () => {
  test('Matches snapshot', () => {
    const tree = renderer.create(
      <Router>
        <HomePage />
      </Router>,
    );
    expect(tree).toMatchSnapshot();
  });
});
