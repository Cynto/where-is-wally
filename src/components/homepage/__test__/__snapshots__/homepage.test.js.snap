// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage tests Matches snapshot 1`] = `
<div
  className="homepage"
>
  <div
    className="button-container"
  >
    <a
      href="/game"
      onClick={[Function]}
    >
      <button
        className="css-button-3d--sky"
      >
        Play Game
      </button>
    </a>
    <a
      href="/leaderboard"
      onClick={[Function]}
    >
      <button
        className="css-button-3d--red"
      >
        Leaderboard
      </button>
    </a>
  </div>
</div>
`;
